.github/workflows/tests.yml

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
You should be able to replace it as [here](https://github.com/inveniosoftware/invenio-records-resources/blob/master/.github/workflows/tests.yml#L32).